### ISSUE TITLE
Cloud formation deploy failure

### ISSUE THREAD

**Issue by innomatics** [aws-solutions/distributed-load-testing-on-aws#111]
**Describe the bug**
Deploying the stack an AWS account that previously hosted the v2 solution, but otherwise is empty resulted in CloudFormation deployment failure.

**To Reproduce**
Install the v2 stack in a fresh AWS account
Delete the v2 stack
Install the current v3 stack

**Expected behavior**
The v3 stack installs

**Please complete the following information about the solution:**
- [x] Version: v3.1.0
- [x] Region: ap-southeast-2
- [x] Was the solution modified from the version published on this repository? No
- [x] If the answer to the previous question was yes, are the changes available on GitHub? N/A
- [x] Have you checked your service quotas? N/A
- [x] Were there any errors in the CloudWatch Logs? N/A

**Additional context**
```
DLTEcsDLTCloudWatchLogsGroupECSLogSubscriptionFilterC5BB4DB5 | CREATE_FAILED | Resource handler returned message: "Could not execute the lambda function. Make sure you have given CloudWatch Logs permission to execute your function. (Service: CloudWatchLogs, Status Code: 400, Request ID: <removed>)" (RequestToken: <removed>, HandlerErrorCode: InternalFailure)
-- | -- | --
```

**Comment by dimitri-lopez** [comment]
This is a known problem within the AWS CDK. See the github [here](https://github.com/aws/aws-cdk/issues/21941) for more details. A workaround was put in place internally for DLT not two days ago and will be available during the next release.

Redeploying the stack should allow you to bypass the issue. There is a roughly 1/5 chance that you will run into this error regardless of region. If you find yourself running into this error a lot we recommend implementing and deploying one of the hot fixes described in the issue above. Even quicker and easier is modifying the DLT template to include:
```
    DependsOn:
      - DLTEcsDLTCloudWatchLogsGroupECSLogSubscriptionFilterCanInvokeLambdaXXXXXXXX
```
under the `DLTEcsDLTCloudWatchLogsGroupECSLogSubscriptionFilterXXXXXXXX` element. Deploying the modified template will stop you from running into issues in the future.

**Comment by gsingh04** [comment]
fixed in `v3.1.1`
